featuretools/tests/computational_backend/test_feature_set_calculator.py

```diff
@@ -1172,12 +1173,30 @@ def test_returns_order_of_instance_ids(pd_es):
 
 def test_calls_progress_callback(es):
     # call with all feature types. make sure progress callback calls sum to 1
+    class CountWithAggType(AggregationPrimitive):
```

Comment:
we should make a separate test instead that checks if a deprecation warning is triggered